src/main/java/org/mule/service/http/impl/service/server/grizzly/GrizzlyServerManager.java

```diff
@@ -80,6 +80,8 @@
 
   private static final long DISPOSE_TIMEOUT_MILLIS = 30000;
 
+  private static final Integer DEFAULT_READ_TIMEOUT_MILLIS = 30000;
```

Comment:
This value has to be the same as the current value. Is it?

Reply:
Yes, it's the same as set on the constructor of AbstractTransport creating the TCPNIOTransport:
![image](https://user-images.githubusercontent.com/16308152/133498102-df674497-096c-4a36-b3f3-d92e1cd82b79.png)